src/docs/releases/1.8.0.md

```diff
@@ -77,8 +77,8 @@ public class ToggleThemeNavbarDisplayDriver : DisplayDriver<Navbar>
     {
         return Task.FromResult<IDisplayResult>(
             View("ToggleTheme", model)
-            .Location("Detail", "Content:10")
-            .Location("DetailAdmin", "Content:10")
+            .Location(OrchardCoreConstants.DisplayType.Detail, "Content:10")
```

Comment:
This constant probably did not existing in 1.8

Reply:
I will check